bin/crank

```diff
@@ -1,8 +1,6 @@
-#!/bin/bash
-# HUH? https://www.cyberciti.biz/tips/finding-bash-perl-python-portably-using-env.html
+#! /usr/bin/env bash
```

Comment:
There's typically no space between shebang and command.

I not a big fan of using

```#!/usr/bin/env bash```

rather than 

```
#!/bin/bash
```

is a good idea.

It's a FANTASTIC idea for anything that isn't a standard part of any unix.

I've *never* seen an issue with directly invoking bash on OS X and Linux.